### ISSUE TITLE
Yobit: InvalidNonce handler doesn't catch that exception

### ISSUE THREAD

**Issue by Djusk8** [ccxt/ccxt#5307]
- OS: MacOS 10.14.3
- Programming Language version: Python 3.6
- CCXT version: 1.18.715
- Exchange: YoBit
- Method: except ccxt.InvalidNonce

I can't catch "Invalid Nonce" exception with  `except ccxt.InvalidNonce`

My code:
```
                try:
                    ex.verbose = True  # todo delete
                    trade_hist = ex.fetch_my_trades(pair, timestamp)
                except ccxt.InvalidNonce:
                    print("Invalid Nonce Exception")
                except ccxt.ExchangeError:
                    print("Exchange Error Exception")
```

Verbose output:
```
Request: GET https://yobit.net/api/3/ticker/dash_btc {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET https://yobit.net/api/3/ticker/dash_btc 200 {'Date': 'Sun, 16 Jun 2019 11:16:47 GMT', 'Content-Type': 'text/html; charset=utf8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=dcc95e0f06a19ad649625967041d419ea1560683807; expires=Mon, 15-Jun-20 11:16:47 GMT; path=/; domain=.yobit.net; HttpOnly; Secure, locale=en; expires=Sat, 16-Jun-2029 11:16:47 GMT; path=/', 'Vary': 'Accept-Encoding', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block;', 'x-download-options': 'noopen', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e7c5fa41a7e9063-DME', 'Content-Encoding': 'gzip'} {"dash_btc":{"high":0.01817228,"low":0.01723438,"avg":0.01770333,"vol":641.79741035,"vol_cur":35788.19295493,"last":0.01779539,"buy":0.01728808,"sell":0.01779627,"updated":1560683584}}

Request: POST https://yobit.net/tapi {'Content-Type': 'application/x-www-form-urlencoded', 'Key': 'xxxxxxxxxxxx', 'Sign': 'xxxxxxxxxxxx', 'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36', 'Accept-Encoding': 'gzip, deflate'} nonce=1560683807&method=TradeHistory&pair=dash_btc&since=0

Response: POST https://yobit.net/tapi 200 {'Date': 'Sun, 16 Jun 2019 11:16:47 GMT', 'Content-Type': 'text/html; charset=utf8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=dcc95e0f06a19ad649625967041d419ea1560683807; expires=Mon, 15-Jun-20 11:16:47 GMT; path=/; domain=.yobit.net; HttpOnly; Secure, locale=en; expires=Sat, 16-Jun-2029 11:16:47 GMT; path=/', 'Vary': 'Accept-Encoding', 'X-Frame-Options': 'SAMEORIGIN', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block;', 'x-download-options': 'noopen', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '4e7c5fa5fb7c9063-DME', 'Content-Encoding': 'gzip'} {"success":0,"error":"invalid nonce (has already been used)"}

Exchange Error Exception
```

It is also written in the documentation that _Invalid Nonce exception_ relates to _Network Error_, but for the YoBit exchange I was able to catch it only with the _Exchange Error_

**Comment by kroitor** [comment]
@Djusk8 this is fixed in 1.18.717, it will arrive in 10 minutes. Let us know if you have any other difficulties with it. Thx for reporting!

**Comment by Djusk8** [comment]
Works fine. Thanks!
